packages/cli/src/zosfiles/compare/CompareBaseHelper.ts

```diff
@@ -164,7 +164,11 @@ export class CompareBaseHelper {
     public prepareContent(content: string | Buffer): string {
         let contentString = content.toString();
         if(this.seqnum === false) {
-            const seqnumlen = 8;
+            let seqnumlen = 8;
+            /* If Windows, account for the carriage return */
+            if(process.platform === "win32" && content.toString().endsWith("\r\n")){
```

Comment:
Is checking for Windows necessary? It's possible to have a local file with Windows line endings (CRLF) on any OS.

Reply:
I tried commenting out the process check and testing on my Mac with a local file that has CRLF. It seems like the first sequence number still shows up:
```
zowe files compare lf-uss temp.jcl "/tmp/XXX/iefbr14.jcl" --no-seqnum
- Removed
+ Added

- //IEFBR14J JOB  (IZUACCT),'TIMOTHY',CLASS=A                             0
- //STEP1    EXEC PGM=IEFBR14                                             0
+ //IEFBR14J JOB  (IZUACCT),'TIMOTHY',CLASS=A                             
+ //STEP1    EXEC PGM=IEFBR14                                             
```

Reply:
I was thinking about that possibility. I will make the change.